ssh_manager.py

```diff
@@ -0,0 +1,122 @@
+from pydbus import SystemBus
+from log import MeticulousLogger
+import subprocess
+import re
+from config import MeticulousConfig, CONFIG_SYSTEM, ROOT_PASSWORD
+
+logger = MeticulousLogger.getLogger(__name__)
+
+
+class SSHManager:
+    ISSUE_PATH = "/etc/issue"
+
+    @staticmethod
+    def set_ssh_state(enabled: bool) -> bool:
+        """
+        Enable or disable SSH service using systemd D-Bus interface.
+        """
+        try:
+            bus = SystemBus()
+            systemd = bus.get(".systemd1")
+
+            if enabled:
+                systemd.EnableUnitFiles(["ssh.service"], False, False)
+                systemd.StartUnit("ssh.service", "fail")
+                logger.info("SSH service enabled and started")
+            else:
+                systemd.StopUnit("ssh.service", "fail")
+                systemd.DisableUnitFiles(["ssh.service"], False)
+                logger.info("SSH service stopped and disabled")
+
+            systemd.Reload()
+            return True
+        except Exception as e:
+            logger.error(f"Error while managing SSH service: {e}")
+            return False
+
+    @staticmethod
+    def handle_manufacturing_mode_exit():
+        manufacturing_config = MeticulousConfig.get("manufacturing", {})
+
+        if not manufacturing_config.get("enable", True) and manufacturing_config.get(
+            "first_boot", True
+        ):
+            logger.info("Detected exit from manufacturing mode and generating password")
+            if SSHManager.generate_root_password():
+                manufacturing_config["first_boot"] = False
+                MeticulousConfig["manufacturing"] = manufacturing_config
+                MeticulousConfig.save()
+                logger.info("Root password generated successfully")
+            else:
+                logger.error("Root password generation failed")
+
+        current_password = SSHManager.get_root_password()
+        if current_password:
+            SSHManager.update_issue_file(current_password)
+            logger.info("Updated /etc/issue with current root password")
+
```

Comment:
we should probably set the root password again here as well. think of the following scenario:
Boot root_a in dev-mode -> password is root
Boot root_b , leave dev mode -> password is changed
Revert to root_a -> Password is root again and will stay root forever